docs/commands/preview.md

```diff
@@ -0,0 +1,67 @@
+# `preview`
+
+## Introduction
+
+The `preview` command starts a local preview server for a Redocly project. Use the preview to develop your project locally before deployment.
+
+{% admonition type="info" name="Important" %}
+This command uses some early-release functionality. Look out for wider release announcements.
+{% /admonition %}
+
+## Usage
+
+```bash
+redocly preview
+redocly preview --product=revel
+redocly preview --product=reef --plan=pro
+redocly preview --product=reef --plan=pro --source-dir=./my-docs-project --port=4001
+```
+
+## Options
+
+| Option           | Type    | Description                                                                                                                                                                                                                                     |
+| ---------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| --help           | boolean | Shows help.                                                                                                                                                                                                                                     |
+| --plan           | string  | Product plan to use in preview. <br/> **Possible values:** `pro`, `enterprise`. The default value is `enterprise`.                                                                                                                              |
+| --product        | string  | Name of a project to preview the product with. <br/> **Possible values:** `redoc`, `revel`, `reef`, `realm`, `redoc-revel`, `redoc-reef`, `revel-reef`. The default value is autodetected from the project's `package.json` or `realm` is used. |
+| --source-dir, -d | string  | Path to the project directory. The default value is `.` (current directory).                                                                                                                                                                    |
+| --port           | number  | The port to run the preview server on. The default value is `4000`.                                                                                                                                                                             |
```

Comment:
Do we support `-p` as well here?

Reply:
No. We have three arguments that start with a `p` - `plan`, `product`, and `port`, so I wasn't sure if it would confuse users.

But considering we have `-p` as short for `port` in other places, it might make sense to add it here as well. What do you think?

Reply:
I think you are right, let's keep the long one as the only option here. We do have port shorthand in the other preview command, but we also have host there, so it's fine to not support it here. And it's easier to add it later than to take it away :)